src/data/community-events.json

```
@@ -1245,6 +1245,14 @@
     "description": "",
     "imageUrl": ""
   },
+  {
+   "title": "ETHDubai",
+   "to": "https://www.ethdubaiconf.org/",
+   "location": "Dubai, UAE",
+   "description": "The Ethereum dev conference and hackathon in Dubai on everything DeFi, privacy, L2s, AVS, AI Agents, intents, Account Abstraction and more with a focus on decentralization and community projects. We also organize a Demo Pitch Day with VCs.",
```

'we' implies ethereum.org are involved, lets just remove this

```suggestion
   "description": "The Ethereum dev conference and hackathon in Dubai on everything DeFi, privacy, L2s, AVS, AI Agents, intents, Account Abstraction and more with a focus on decentralization and community projects.",
```

---

I pushed a new change, let me know if it works for you.